src/main/java/fr/cgi/minibadge/core/constants/Rights.java

```diff
@@ -0,0 +1,14 @@
+package fr.cgi.minibadge.core.constants;
+
+public class Rights {
+    public static final String ASSIGN = "minibadge.assign";
+    public static final String FULLNAME_ASSIGN = "fr.cgi.minibadge.controller.FakeRight|assign";
+    public static final String RECEIVE = "minibadge.receive";
+    public static final String FULLNAME_RECEIVE = "fr.cgi.minibadge.controller.FakeRight|receive";
```

Comment:
Pour être sur du package (si jamais ça doit changer) tu pourrais faire ça 

```suggestion
 public static final String FULLNAME_RECEIVE = String.format("%s.%s|assign", FakeRight.class.getPackage().getName(), FakeRight.class.getSimpleName());
```

Ca t'éviterait de devoir changer le Rights si jamais ca change de package/controller (même si c'est très peu probable mais si on peut éviter d'écrire en magic string "fr.cgi.minibadge.controller" comme on a déjà l'info avec le model Package/Class)

Reply:
Yes ! j'aime beaucoup ! je prend 😉